src/decisionengine/framework/logicengine/LogicEngine.py

```diff
@@ -7,22 +7,21 @@
 from decisionengine.framework.modules.Module import Module
 
 class LogicEngine(Module):
-    def __init__(self, cfg, channel_name):
+    def __init__(self, cfg):
         super().__init__(cfg)
-        self.logger = structlog.getLogger(f"{channel_name}")
         self.logger = self.logger.bind(module=__name__.split(".")[-1])
         self.facts = {name: BooleanExpression(expr) for name, expr in cfg["facts"].items()}
         self.rule_engine = RuleEngine(cfg["facts"].keys(), cfg["rules"])
 
     def produces(self):
-        self.logger.debug("in LE::produces()")
+        self.get_logger().debug("in LE::produces()")
```

Comment:
I'm not finding where `get_logger` is defined.... I didn't see it under `decisionengine.framework.modules.Module` or in this class....